src/encord_active/app/data_quality/common.py

```diff
@@ -35,7 +36,7 @@ def get_metric_operation_level(pth: Path) -> str:
 
 
 @st.experimental_memo
-def load_available_metrics(metric_type_selection: MetricType) -> List[MetricData]:
+def load_available_metrics(metric_type_selection: str) -> List[MetricData]:
```

Comment:
Any reason why this is a string and not the Enum?

Reply:
`@st.experimental_memo` requires all arguments to be hashable. (should be addressed by the big refactor)